examples/values.yaml

```diff
@@ -0,0 +1,90 @@
+--- 
+core:
+  projectId: "gcp-project-id"
+  image: 
+    pullPolicy: IfNotPresent
+    registry: gcr.io/kf-feast
+    repository: feast-core
+    tag: "1234"
+  replicaCount: 3
+  resources: 
+    limits: 
+      cpu: 8
+      memory: 8G
+    requests: 
+      cpu: 2
+      memory: 4G
+  rollingUpdate: 
+    maxSurge: 2
+    maxUnavailable: 0
+  service: 
+    grpc: 
+      port: 6565
+      targetPort: 6565
+    http: 
+      port: 80
+      targetPort: 8080
+  jobs: 
+    runner: DirectRunner
+    options: "{}"
+    errorStoreType: "stdout"
+    errorStoreOptions: "{}"
+    monitoring:
+      period: 5000
+      initialDelay: 60000
+  trainingDatasetPrefix: "fs"
+  # logType: JSON
+
+postgresql: 
+  provision: true
+  imageTag: 9.6.11
+  service: 
+    port: 5432
+    
+redis:
+  name: feast-redis
+  provision: true
+
+serving: 
+  config: 
+    maxEntityPerBatch: 2000
+    maxNumberOfThread: 128
+    redisPool: 
+      maxIdle: 16
+      maxSize: 128
+    timeout: 5
+  image: 
+    pullPolicy: IfNotPresent
+    registry: gcr.io/kf-feast
+    repository: feast-serving
+    tag: "1234"
```

Comment:
you can set this tag to our latest release (0.1.0)

Reply:
Done, thanks @zhilingc